tools/echo_server/echo_server.go

```diff
@@ -58,13 +58,35 @@ type Argument struct {
 	ServerPort 				string `json:"server-port"`
 	ServerCert 				string `json:"server-certificate"`
 	ServerKey  				string `json:"server-key"`
+	ServerCertPath          string `json:"server-certificate-location"`
+	ServerKeyPath           string `json:"server-key-location"`
```

Comment:
Do you mind running the go formatter to enforce the formatting

Reply:
fixed in next rev.